### PR TITLE
Updating the deprovisioning steps

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,11 +142,16 @@ After deleting your cluster deployment you will see an uninstall job created. If
...
  1. Delete the uninstall job, it will be recreated and tried again.
  1. Manually delete the uninstall finalizer allowing the cluster deployment to be deleted, but note that this may leave artifacts in your AWS account.
-    * `kubectl edit clusterdeployments.hive.openshift.io YOURCLUSTER`
- 1. You can manually run the uninstall code with hiveutil to delete AWS resources based on their tags.
-    * Get your cluster UUID from the clusterdeployment.Spec.ClusterUUID.
-    * `make hiveutil`
-    * `bin/hiveutil aws-tag-deprovision --loglevel=debug --cluster-name CLUSTER_NAME openshiftClusterID=CLUSTER_UUID kubernetes.io/cluster/CLUSTER_NAME=owned`
+ 1. You can manually run the uninstall code with `hiveutil` to delete AWS resources based on their tags.
+    * Run `make hiveutil`
+    * Get your cluster tag from the following command output. Look for infraID in status.infraID
```
Comment (dgoodwin) [inline README.md]: This should clarify that if your cluster deployment is gone you can also find the tag in AWS console on almost any cloud object from that cluster.

File: README.md [hunk 2]
```diff
@@ -142,11 +142,16 @@ After deleting your cluster deployment you will see an uninstall job created. If
...
+      ```bash
+      $ kubectl edit clusterdeployments.hive.openshift.io ${CLUSTER_NAME}
```
Comment (dgoodwin) [inline README.md]: kubectl get cd dgoodwin1 -o json | jq -r '.status.infraID'

File: README.md (hunk 1 above)
Comment (LalatenduMohanty) [reply README.md]: Agree

File: README.md [hunk 3]
```diff
@@ -142,11 +142,17 @@ After deleting your cluster deployment you will see an uninstall job created. If
...
-    * `kubectl edit clusterdeployments.hive.openshift.io YOURCLUSTER`
- 1. You can manually run the uninstall code with hiveutil to delete AWS resources based on their tags.
-    * Get your cluster UUID from the clusterdeployment.Spec.ClusterUUID.
-    * `make hiveutil`
-    * `bin/hiveutil aws-tag-deprovision --loglevel=debug --cluster-name CLUSTER_NAME openshiftClusterID=CLUSTER_UUID kubernetes.io/cluster/CLUSTER_NAME=owned`
+ 1. You can manually run the uninstall code with `hiveutil` to delete AWS resources based on their tags.
+    * Run `make hiveutil`
+    * Get your cluster tag from the following command output. Look for infraID in status.infraID
+      ```bash
+      $ kubectl get cd ${CLUSTER_NAME} -o json | jq -r '.status.infraID'
```
Comment (LalatenduMohanty) [inline README.md]: @dgoodwin I have not tested this command yet. But will do that soon.

**Comment by LalatenduMohanty** [comment]
@dgoodwin This is ready to be merged.

**Pull request merged**